nano/node/lmdb/lmdb.cpp

```diff
@@ -691,3 +791,453 @@ bool nano::mdb_store::init_error () const
 {
 	return error;
 }
+
+// All the v14 functions below are only needed during upgrades
+nano::summation_visitor_v14::summation_visitor_v14 (nano::transaction const & transaction_a, nano::mdb_store const & store_a) :
```

Comment:
Difference between summation_visitor_v14 & summation_visitor seems just 2-3 lines (actually call to block_get_v14 instead of block_get). Could we avoid recreating whole visitor 2 times?

Reply:
Yeh this one is much simpler to combine as it doesn't make use of the extra arguments, so I don't think it affects the interface too much.